planner/core/plan_to_pb.go

```diff
@@ -232,6 +239,7 @@ func (p *PhysicalTableScan) ToPB(ctx sessionctx.Context, storeType kv.StoreType)
 			telemetry.CurrentTiflashTableScanWithFastScanCount.Inc()
 		}
 	}
+	// err = tables.SetPBColumnsDefaultValue(ctx, tsExec.Columns, p.Columns)
```

Comment:
Please remove the same line ~

Reply:
I will change this after the pr in tipb merged.